src/cmds/qstat.c

```diff
@@ -1126,10 +1126,12 @@ percent_cal(char *state, char *timeu, char *timer, char *wtimu, char *wtimr, cha
 		case 'Q':
 		case 'T':
 		case 'W':
+			pbs_asprintf(&rtn, "%3s", "-- ");
```

Comment:
I expected to see a corresponding call to free(). I realize that qstat is supposed to run quickly and exit, but we should still be mindful of freeing dynamically allocated memory.

Reply:
It does get freed by the caller of `percent_cal`: https://github.com/PBSPro/pbspro/blob/264cefe83eea3f51f5f84d77cd3a837e8dacbf46/src/cmds/qstat.c#L1430-L1432

Reply:
You are correct. Thanks for pointing it out.